cmd/monaco/integrationtest/v2/loading-report_test.go

```diff
@@ -93,7 +93,7 @@ func TestLoadingReport(t *testing.T) {
 					Type:  report.TypeLoad,
 					Time:  report.JSONTime(time.Now()),
 					State: report.StateError,
-					Error: "cannot parse definition in `invalid-classic-with-settings\\config.yaml`: config api type (notification) configuration can only reference IDs of other config api types - parameter \"alertingProfileId\" references \"builtin:alerting.profile\" type",
+					Error: "config api type (notification) configuration can only reference IDs of other config api types - parameter \"alertingProfileId\" references \"builtin:alerting.profile\" type",
```

Comment:
Removed the start of it
Reason:
- It's anyway a contains match
- This would need some adjustments (`\` vs `/` on different operating systems)